prestashopv1/plugin/modules/indodana/indodana.php

```diff
@@ -41,7 +41,7 @@ public function __construct()
     {
         $this->name = 'indodana';
         $this->tab = 'payments_gateways';
-        $this->version = '0.1.0';
+        $this->version = '1.0.0';
```

Comment:
I think it's better to stay `0.1.0` atm because we still haven't UAT and stuffs. After all is done, we'll change this to `1.0.0` before releasing the plugin publicly

Reply:
Okay, reverted back to version 0.1.0